### PR TITLE
add tauri example; add install guide

### PR DESCRIPTION
**Pull request by awkj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ Make your windows vibrant.
 - `apply_mica()` & `clear_mica()` - **`Windows 11`**
 - `apply_vibrancy()` - **`macOS 10.10+`**
 
+## Install Guide
+
+in `Cargo.toml`  add the content
+
+```toml
+[dependencies]
+window-vibrancy = { git = "https://github.com/tauri-apps/window-vibrancy", branch = "dev" }
```
Comment (amrbashir) [inline README.md:25]: The crate is now published, no need to use it from github anymore.

File: examples/tauri.rs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+      #[cfg(target_os = "windows")]
+      apply_blur(&window, Some((18, 18, 18, 125)))
+        .expect("Unsupported platform! 'apply_blur' is only supported on Windows");
+
+      Ok(())
+    })
+    .run(tauri::generate_context!())
+    .expect("error while running tauri application");
+}
```
Comment (amrbashir) [inline examples/tauri.rs:26]: This example won't compile. Instead, you should include a full tauri project i.e one where `tauri.conf.json` and `Cargo.toml` is present.

**Review by amrbashir — CHANGES_REQUESTED** [review]
Adding the tauri example is fine but the installation instructions is not necessary, it is now published to crates.io .

File: examples/tauri.rs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  tauri::Builder::default()
+    .setup(|app| {
+      let win = app.get_window("main").unwrap();
+
+      #[cfg(target_os = "macos")]
+      apply_vibrancy(&win, NSVisualEffectMaterial::AppearanceBased)
+        .expect("Unsupported platform! 'apply_vibrancy' is only supported on macOS");
+
+      #[cfg(target_os = "windows")]
+      apply_blur(&window, Some((18, 18, 18, 125)))
```
Comment (doonv) [inline examples/tauri.rs:19]: `window` isn't defined. I'm pretty sure you meant to use `win` right?
```suggestion
      apply_blur(&win, Some((18, 18, 18, 125)))
```

Comment (awkj) [reply examples/tauri.rs:19]: Yes, I only have macos, sorry , I forget this PR, wait me , let me fix it and add more code in  example

**Comment by amrbashir** [comment]
thank you, I have added an example in https://github.com/tauri-apps/window-vibrancy/commit/648bf33d26a6301e6cfcc9bc0c83d8eebe67b18d

**Pull request closed without merging**